_posts/2020-06-21-ski-numerals.markdown

```diff
@@ -0,0 +1,923 @@
+However, most of the time, you can go find something for which the exact number is not known, and go bound it for the first time, or improve an existing bound, with little effort.
+Congratulations, you just discovered a new mathematical fact and ever so slightly expanded humanity's collection of useful information.
+
+Even if a fact seems easy to prove and not very useful to the discoverer, I would say that alone does not inherently make it unworthy of publication.
+Please do share your small discoveries.
+Once discovered and published, it becomes a fact that does not need to be derived again.
+It may become a useful stepping stone for a larger research work.
+That path may have yet to be mapped out.
+That research work may not have been started yet.
+The field of that research may not even exist yet.
+When we do get there though, we will be one step closer to the goal than if you had not shared your discovery.
+
+That is why I am now publishing some lower bounds I have found for the \\\(\Xi\\\) function, and all the rough work in deriving them.
+Even though these seem to be a trivially obvious improvement over [Lawrence Hollom's bounds](https://sites.google.com/a/hollom.com/extremely-big-numbers/home/xi),
+which are the best I know of other than my own work,
+and even if I have no use for these discoveries,
+I still believe these results are interesting enough to share, and I hope, even if they are not directly useful in serious mathematics, that I can at least alleviate some first steps or provide inspiration on where to go.
+
+</div>
+
+# Definitions
+{: #definitions}
+
+{% include collapser.markdown %}
+
+<div>
+
+The SKI combinator calculus is a very minimal computational system.
+You should read about it on [its Wikipedia article](https://en.wikipedia.org/wiki/SKI_combinator_calculus) if you're unfamiliar with the topic.
+
+I'm not going to explain it all again, but in short, it is like a programming language whose only objects are unary functions, and whose only primitives are the combinators \\\(S, K, I\\\), hence the name.
+As a reminder, since there are only unary functions, the brackets are often dropped except when needed to show order of evaluation, and expressions are read left to right.
+For example, \\\(SKS(KI)S\\\) is a shorthand for the following pseudocode program:
+
+```
+     a ← S(K)
+     b ← a(S)
+     c ← K(I)
+     d ← b(c)
+result ← d(S)
+```
+
+As a reminder, this is what each combinator does:
+
+```
+Sxyz = xz(yz)
+Kxy = x
+Ix = x
+```
+
+What we informally call "evaluation" of these rules is formally called "beta-reduction".
+One would beta-reduce an SKI expression until no further rules can be applied,
+which occurs when none of the combinators have enough arguments to use their rule.
+Conventionally, we beta-reduce at the leftmost possible sub-expression.
+For example, starting with \\\(SSS(SI)S\\\), here is a the order of steps until it is maximally beta-reduced:
```

Comment:
"here is the order of steps" (taking out the extra "a")

Reply:
good catch!